src/handlers/summary.ts

```diff
@@ -29,9 +29,9 @@ function generateGitHubSummary(context: Context, urls: ResultInfo[]): string {
   output.push("\n</samp>\n");
   output.push("## Configuration\n\n");
   output.push("### Watching Repositories\n\n");
-  output.push(context.config.repos.monitor.map((o) => `- [${o}](${target}/${o})`).join("\n"));
+  output.push(context.config.repos?.monitor.map((o) => `- [${o}](${target}/${o})`).join("\n"));
```

Comment:
Or else you get situations like this

Reply:
`repos` will never be undefined during runtime.